package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "dev": "next dev -p 3000",
     "dev-local": "set \"TARGET_ENV=development-localserver\" && next dev -p 3000",
+    "dev-mock": "MOCKING=true next dev -p 3000",
```

Comment:
I wasn't able to get this to work. I had to modify it to `"dev-mock": "set \"MOCKING=true\" && next dev -p 3000",`

Am I missing something?

Fwiw, I'm on a windows machine.

Reply:
I changed it now by adding `cross-env`. That's a problem between different OS platforms. But with this little tool, it should work on all OS platforms.

Reply:
Works like a champ 👍